projects/eleven-finance/index.js

```diff
@@ -0,0 +1,194 @@
+const sdk = require('@defillama/sdk')
+const abi = require('./abi.json')
+const BigNumber = require('bignumber.js')
+
+const bsc_dashboard = "0x1ac6C0B955B6D7ACb61c9Bdf3EE98E0689e07B8A"
+const bsc_pools = [
```

Comment:
Is it possible to get these pools from an API or an external source that will stay up to date?

Reply:
I just talked with the team and it looks like no. They do post in a telegram announcements channel when a new pool is added, so I may be able to put together something that updates the list when they announce a new pool. They don't post the addresses though, so I need to make sure I can find a way to scrape the address from the platform when I know there's a new one.

Reply:
Actually never mind, a different team member got back to me and there is. Going to close this PR and integrate that.

Reply:
Nice, a lot of our adapters have ended up with hardcoded addresses and maintaining these is really bothersome (and I always end up forgetting) so I try to avoid it as much as possible :)